docs/source/data-model.rst

```diff
@@ -237,10 +354,16 @@ can always be losslessly transformed into an Event and vice versa.
 Run Stop Document
 -----------------
 
+A 'stop' document marks the end of the run. It contains metadata that is not
+known until the run completes.
+
+The most commonly useful fields here are 'time' and 'exit_status'.
+
 Minimal nontrivial valid example:
 
 .. code-block:: python
 
+   # 'stop' document
```

Comment:
Do you think we should add a comment like this to _all_ the examples? Most of them do not have one. I would rather apply it to none or all.

Reply:
I added them to all the examples.